tables/competitions_extra.sql

```diff
@@ -95,6 +97,7 @@ SELECT
 	e.timezone_id timeZoneId,
 	a.announced_at announcedAt,
 	a.results_posted_at resultsPostedAt,
+	IF(DATEDIFF(a.start_date,CURRENT_DATE) >0, 1, 0) upcoming,
```

Comment:
a.start_date and CURRENT_DATE should be switched the other way, and I prefer using CURDATE()